packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/CurrencyLayouts/LayoutUsdArs/index.tsx

```diff
@@ -0,0 +1,255 @@
+                </Title>
+                <Value data-e2e='sbCbu'>{agent.address}</Value>
+              </div>
+            </RowCopy>
+          ) : (
+            <RowCopy>
+              <div>
+                <Title>
+                  <FormattedMessage
+                    id='modals.simplebuy.transferdetails.cvu'
+                    defaultMessage='CVU'
+                  />
+                </Title>
+                <Value data-e2e='sbCvu'>{agent.address}</Value>
+              </div>
+            </RowCopy>
+          )}
+          <RowCopy>
+            <div>
+              <Title>
+                <FormattedMessage
+                  id='modals.simplebuy.transferdetails.account'
+                  defaultMessage='Account Number'
+                />
+              </Title>
+              <Value data-e2e='sbAccountNumber'>{props.account.agent.code}</Value>
+            </div>
+          </RowCopy>
+          <RowCopy>
+            <div>
+              <Title>
+                <FormattedMessage
+                  id='modals.simplebuy.transferdetails.recipient'
+                  defaultMessage='Recipient'
+                />
+              </Title>
+              <Value data-e2e='sbRecipientName'>{props.account.agent.recipient}</Value>
+            </div>
+          </RowCopy>
+        </ExpansionPanel>
+      </div>
+      <Bottom>
+        <BottomInfoContainer>
+          <BottomRow>
+            <DisplayIcon>
+              <Icon size='18px' color='grey800' name='bank-filled' />
+            </DisplayIcon>
+            <BottomMultiRowContainer>
+              <DisplayTitle>
+                <FormattedMessage
+                  id='modals.simplebuy.deposit.bank_transfer_only'
+                  defaultMessage='Bank Transfers Only'
+                />
+              </DisplayTitle>
+              <DisplaySubTitle>
+                <FormattedMessage
+                  id='modals.simplebuy.deposit.bank_transfer_only_description_ars'
+                  defaultMessage='Only send funds from a bank account in your name. If not, your deposit could be rejected.'
+                />
+              </DisplaySubTitle>
+            </BottomMultiRowContainer>
+          </BottomRow>
+          <BottomRow>
+            <DisplayIcon>
+              <Icon size='18px' color='grey800' name='pending' />
+            </DisplayIcon>
+            <BottomMultiRowContainer>
+              <DisplayTitle>
+                <FormattedMessage
+                  id='modals.simplebuy.deposit.processing_time'
+                  defaultMessage='Processing Time'
+                />
+              </DisplayTitle>
+              <DisplaySubTitle>
+                <FormattedMessage
+                  id='modals.simplebuy.deposit.processing_time.info.usdars1'
+                  defaultMessage='Funds will be credited to your {currency} wallet as soon as we receive them. Funds are generally available within 3 business days. We will send you a mail with the confirmation.'
```

Comment:
`We will send you a mail with the confirmation.` should be `We will send you an email with the confirmation.`